pkg/vex/vex.go

```diff
@@ -58,12 +58,19 @@ type Metadata struct {
 
 	// Timestamp defines the time at which the document was issued.
 	Timestamp *time.Time `json:"timestamp"`
-}
 
-// VulnerabilityReference captures other identifier assigned to the CVE.
-type VulnerabilityReference struct {
-	RefType   string `json:"type"` // URL, OSV, FEDORA, etc
-	Reference string `reference:"ref"`
+	// Version is the document version. It must be incremented when any content
+	// within the VEX document changes, including any VEX statements included within
+	// the VEX document.
+	Version string `json:"version"`
```

Comment:
@puerco Would it make more sense for this field's type to be `int` instead?

Reply:
No, that would exclude out a lot of versioning schemes like semver, etc